spec/features/geo_viewer_spec.rb

```diff
@@ -17,14 +17,23 @@
     it 'shows the map controls' do
       expect(page).to have_css('.leaflet-control-zoom-in', count: 1, visible: true)
       expect(page).to have_css('.leaflet-control-zoom-out', count: 1, visible: true)
-      
-      within '.leaflet-control-attribution.leaflet-control' do 
+
+      within '.leaflet-control-attribution.leaflet-control' do
         expect(page).to have_content(' contributors, Tiles Courtesy of ')
       end
     end
 
     it 'shows the Mapquest Tile' do
-      expect(page).to have_css "img[src*='mqcdn.com']", count: 8
+      expect(page).to have_css("img[src*='mqcdn.com']", minimum: 6)
+    end
+
+    it 'shows the bounding box' do
+      # This is a hack, but there is no other way (that we know of) to
+      # find this svg element on the page.
+      # We also need to explicitly wait for the JS to run.
+      find ".leaflet-overlay-pane"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.